src/renderer/components/AircraftSection/index.tsx

```diff
@@ -203,6 +206,21 @@ const index: React.FC<Props> = (props: Props) => {
         }
     }
 
+    function getLastInstalledBranch(): string {
+        const branchName = settings.get('cache.' + props.mod.key + '.lastInstalledBranchName');
+        const branchKey = settings.get('cache.' + props.mod.key + '.lastInstalledBranchKey');
+        if (typeof branchName === "string") {
```

Comment:
**suggestion:** If you make sure that the data injected in the `settings` object is always a string so no need to check for the type each time.